README.md

```diff
@@ -756,6 +756,33 @@ me to take it over and continue working on the project. That helped to revive it
 
 ## Release Notes
 
+* 5.3.3
```

Comment:
Maybe instead of writing change log inside the readme, we should create a dedicated file `CHANGELOG.md`?

Reply:
Great suggestion. I added it.